homeassistant/components/arcticspa/config_flow.py

```diff
@@ -0,0 +1,94 @@
+"""Config flow for Arctic Spa integration."""
+from __future__ import annotations
+
+from http import HTTPStatus
+import logging
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_API_KEY, CONF_NAME
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import CONF_ARCTIC_SPA_KEY_NAME, DOMAIN
+from .hottub import Device
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_NAME, default=CONF_ARCTIC_SPA_KEY_NAME): str,
```

Comment:
We only allow helpers to set names in the config flow.

Reply:
Noted, updated.